pkgs/desktops/cinnamon/cinnamon-gsettings-schemas/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
```

Comment:
preferably, you should name this `cinnamon-gsettings-overrides`. The name of the package in pantheon isn't correct and confusing.